microshift_updating/microshift-about-updates.adoc

```diff
@@ -0,0 +1,46 @@
+:_content-type: ASSEMBLY
+[id="microshift-about-updates"]
+=  About {product-title} updates
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-about-updates
+
+toc::[]
+
+You can update a {product-title} cluster by using the OpenShift CLI (`oc`).
+// This PR is for the book build. Note that the OCP structure consists of a landing page of xrefs to other major sections within the book. MicroShift likely does not require that depth of structure, so starting simply with one page.
+
+[id="microshift-about-updates-understanding-microshift-updates"]
+== Understanding {product-title} updates
+{product-title} updates are available as either RPMs or by embedding the {product-title} image in an RPM OSTree blueprint.
+You can update an {product-title} cluster by using the OpenShift CLI (`oc`).
+//Platform administrators can view new update options by looking at the output of the `oc adm upgrade` command.
+//An update begins when...
+
+[NOTE]
+====
+Operators previously installed must be reinstalled using manifests.
+====
+
+=== Using RPM updates
+Using the RPM update method replaces your existing version. No rollback is possible with this update type.
+//we can call a module here or xref out; not sure the best method for our use case until we have the content
+
+=== Using RPM OSTree updates
+Using the RPM OSTree update path allows for system rollback.
+//we can call a module here or xref out; not sure the best method for our use case until we have the content
+
+[id="microshift-about-updates-checking-version-update-compatibility"]
+== Checking version update compatibility
+Before attempting an update, determine which version of {product-title} you have installed. Only the following update paths are supported:
+
+* Version 4.13 to 4.14
+//replace with matrix including RHEL versions?
+//place xref here to version-check assembly
+
+[id="microshift-about-updates-update-disconnected-environment"]
+== Updating a cluster in a disconnected environment
```

Comment:
This would likely be its own module too, in the other assembly specifically for update procedures? If there is such another assembly.